docs/deploying-open-commerce.md

```diff
@@ -0,0 +1,41 @@
+# Deploying Open Commerce
+
+Open Commerce is an open source eCommerce platform suitable for use in product. All services are either publicly available Docker images on DockerHub or are Git repositories that include a `Dockerfile` so that you can build your own Docker image after customizing the service. This means that you can deploy the Open Commerce system on any infrastructure that supports running a Docker container cluster with a shared network.
+
+## Deploying Open Commerce on Digital Ocean
+
+The following guide is intended for users that are in the evaluation phase of Open Commerce. It is not meant to be a production
+grade deployment, rather a simple guided tutorial that will provide insight on how the platform works together. With that said,
+the concepts covered in the guided can be used to build a production grade deployment.
+
+[Deploying Open Commerce on Digital Ocean using Ansible and Traefik](https://github.com/reactioncommerce/proxy-traefik)
+
+## Deploying Open Commerce with Kubernetes via Helm
+
+[Kubernetes](https://kubernetes.io/) is a technology created by Google for orchestrating containers in production. While Kubernetes
+(often referred to as K8) is a powerful tool, it can also be complex and daunting. To make this easier Open Commerce offers Helm
```

Comment:
```suggestion
(often referred to as K8) is a powerful tool, it can also be complex and daunting. To make this easier Open Commerce offers `Helm
```

Reply:
Unclear why we are putting "Helm Charts" in quotes?